### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by viatrix**

### CODE REVIEW

File: test/debts.js [hunk 1]
```diff
@@ -47,13 +47,71 @@ contract('Debts', function(accounts) {
...
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.borrow(value, {from: borrower}))
+    .then(() => debts.repay(borrower, value, {from: borrower}))
+    .then(() => debts.debts(borrower))
+    .then(asserts.equal(value));
+  });
 
-  it('should not allow not owner to repay');
+  it('should fail on overflow when repaying', () => {
```
Comment (lastperson) [inline test/debts.js:100]: What about the overflow when borrowing?

File: test/debts.js [hunk 2]
```diff
@@ -47,13 +47,71 @@ contract('Debts', function(accounts) {
...
+  it('should allow to borrow' , () => {
+      const borrower = accounts[3];
+      const value = 1000;
+      return Promise.resolve()
+      .then(() => debts.borrow(value, {from: borrower}))
+      .then(() => debts.debts(borrower))
+      .then(asserts.equal(value));
+  });
+
+  it('should emit Repayed event on repay', () => {
```
Comment (lastperson) [inline test/debts.js:59]: Is there any other events in the contract?

**Review by lastperson — CHANGES_REQUESTED** [review]
Is there any other cases? Will multiple different borrowers work alright? Maybe need to check other getters of the contract?

File: test/debts.js (hunk 2 above)
Comment (viatrix) [reply test/debts.js:59]: 'Borrowed' event was already checked in 'should emit Borrowed event on borrow' test

File: test/debts.js (hunk 1 above)
Comment (viatrix) [reply test/debts.js:100]: This case was already checked in 'should fail on overflow when borrowing' test

Comment (lastperson) [reply test/debts.js:100]: right, my bad, was only looking at the changes.

File: test/debts.js (hunk 2 above)
Comment (lastperson) [reply test/debts.js:59]: Same, forgot about current ver.

**Review by lastperson — COMMENTED** [review]
I like it!

**Review by lastperson — APPROVED** [review]

**Pull request closed without merging**